cryptoservice/import_export_test.go

```diff
@@ -199,7 +199,7 @@ func TestImportExportGUN(t *testing.T) {
 		if alias == data.CanonicalRootRole {
 			continue
 		}
-		relKeyPath := filepath.Join("tuf_keys", privKeyName+".key")
```

Comment:
Non-blocking nitpick:  can we also use the `notary.NonRootKeysSubdir` constant here and below?